Autopilot/PathManager/Src/pathStateClasses.cpp

```diff
@@ -90,16 +231,79 @@ pathManagerState& cruisingState::getInstance()
     return singleton;
 }
 
+
 void coordinateTurnElevation::execute(pathManager* pathMgr)
 {
+    
+    SFOutput_t * sensFusionOutput = sensorFusion::GetSFOutput(); // Get sensor fusion data
+    IMU_Data_t * imudata = sensorFusion::GetIMUData(); // Gets raw IMU data
+    _WaypointManager_Data_Out * waypointOutput = cruisingState::GetOutputData(); // Get output data from waypoint manager
+    
+    CoordinatedTurnInput_t turnInput {};
+    AltitudeAirspeedInput_t altAirspeedInput {};                                   
+
+    
     //get elevation and turning data
-    if(isError)
+    //loading in commands data from each of the states
+    switch(pathMgr->stage){
+        case ROLL:
+            LandingManager::translateLTSFCommandsToCoordTurns(takeoffRollStage::getControlOutput(), sensFusionOutput, imudata, &turnInput, &altAirspeedInput);
+            break;
+        case CLIMB:
+            LandingManager::translateLTSFCommandsToCoordTurns(takeoffClimbStage::getControlOutput(), sensFusionOutput, imudata, &turnInput, &altAirspeedInput);
```

Comment:
Climb is being taken care of by the landing manager ? That's very unintuitive

Reply:
The function for translating both landing and takeoff outputs to coord turn commands lives in the landing manager class. Should I make it a stand alone function?

Reply:
I'll take a closer look and lyk, this may be the best solution, but it feels weird

Reply:
Ok I see. Really it's the landingTakeoff manager, no problem. I think changing the name to that would make things a bit clearer. THoughts ?